### PR TITLE
Adapt saturn-theme to use @frontity/h2r package

### PR DESCRIPTION
**Pull request by DAreRodz**
Fixes https://github.com/frontity/frontity/issues/76

### CODE REVIEW

File: src/shared/processors/low/wpappbox.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (orballo) [inline src/shared/processors/low/wpappbox.js]: Did these components need any change? If positive, had all of them been tested?

If these components are working like before and just moved the code to the new folder, don't mind this comment.

**Comment by luisherranz** [comment]
@orballo @david, is this ready to merge? I need it for the release.

**Comment by luisherranz** [comment]
@DAreRodz @orballo I still need this

**Comment by orballo** [comment]
@luisherranz I think @DAreRodz was going to write some tests for the different processors, because not all of them had been tested.

**Comment by luisherranz** [comment]
Ok, then please my question so I can know if I can do the deploy today or not :)

**Comment by orballo** [comment]
@DAreRodz let me know when I should review this PR again :)

**Comment by luisherranz** [comment]
I've moved this PR back to "In progress" because @DAreRodz is working on it.

Once it's done, @DAreRodz should move it back to "In Review" again.

**Comment by orballo** [comment]
Perfect :smile:

**Comment by luisherranz** [comment]
Awesome, thanks guys!

**Pull request merged**